justfile

```diff
@@ -50,8 +48,29 @@ _pre-run:
         exit 1
     fi
 
-# Run examples
-run-examples *FLAGS: _pre-build _pre-run
+# Build only the zkVM ELF program in release mode (optimized)
+build-elf:
+    #!/usr/bin/env bash
+    if ! {{ path_exists(cargo-prove-path) }}; then 
+        echo -e "⛔ Missing zkVM Compiler.\nRun \`just initial-config-installs\` to prepare your environment"; 
+        exit 1; 
+    fi; 
+    source {{ env-settings }}; 
+    if [ ! -f "$ZK_PROGRAM_ELF_PATH" ]; then 
+        echo -e "Can't find ELF at \`$ZK_PROGRAM_ELF_PATH\`.\nAttempting to build it..."; 
+        RUSTFLAGS="-Copt-level=3 -Clto=fat -Ccodegen-units=1 -Cdebuginfo=1 -Cembed-bitcode=yes" cargo prove build -p eq-program-keccak-inclusion 
+    else 
+        echo "✅ ELF Exists, skipping SP1 build"; 
+    fi
```

Comment:
this only compiles the ELF with optimizations, so the Cargo.toml isn't technically needed, but "safe" to be sure we don't forget it / miss in CI.